### PR TITLE
Retrofit Egyptian pagansim from CK2+ 3.05.4

### PR DESCRIPTION
**Pull request by rquinio**
Hi xirsoi,
I've retrofited egyptian_pagan religion from CK2+, with permission from schwarherz.
I've play-tested all the decisions and associated events, works fine ! 
Also no additional validation errors detected by the Validator ;)

I made some minor adjustments:
- Rename/split files using ACR_ prefix / ACR sub-folder
- Adapt egyptian_pagan religion icon number to ACR strip
- Fix some decisions images frame
- Replace Luxor event image with less cartoonish Egyptian temple image
- Switch egyptian_pagan graphical culture to africangfx, as persiangfx doesn't exist
- Update README to list contribution from schwarherz / CK2+

Note: there are no event for rebirth of Egyptian pagansim for now, so usage of console / ruler designer is required.

### CODE REVIEW

File: AncientReligions/events/ACR_egyptian_events.txt [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+			}
+		}		
+	}	
+}
+
+# The Priesthood Restored
+narrative_event = {
+	id = ancrel.4006
+	title = EVTNAME_ancrel_4006
+	desc =�EVTDESC_ancrel_4006
```
Comment (xirsoi) [inline AncientReligions/events/ACR_egyptian_events.txt]: Do you know what this question mark is?

**Comment by xirsoi** [comment]
I'm thinking we should form a team, if you're interested that is.
You and me, plus jbelig (from the ck2 forums). He seems quite good at making Coats of Arms and is enthusiastic. He was also helpful in nailing down some bugs.

File: AncientReligions/events/ACR_egyptian_events.txt (hunk 1 above)
Comment (rquinio) [inline AncientReligions/events/ACR_egyptian_events.txt]: Nice spot, must have been a bad copy//pasted white-space character

**Comment by rquinio** [comment]
Sure, we need to keep this mod alive !
Now that WtWSMS mod is out, it also gives a nice setup in 480 A.D. to play these ancient religions, and it relies on ACR for the religious flavor event/decisions. So there should be plenty of work to do :)

**Comment by xirsoi** [comment]
To begin with, we should almost never work directly on master.
Master is the version that should always be working, the version that users can always download from and expect it to Just Work.

Work in a separate branch and submit your pull requests to develop so we can properly test them before making a release.

**Comment by rquinio** [comment]
Well I don't really agree on users downloading directly from master ^^
It makes managing bug reports difficult (need to know WHEN or which commit they downloaded, bug may already have been fixed, etc.) Fixed version via git tags with changelog is safer IMO.

Are you ok to use github issues to track and discuss bugs/changes ?

**Comment by xirsoi** [comment]
That makes a lot of sense, we can do it that way.

I just sent you a pm wondering how we should discuss the mod. github issues sounds like a great way.

**Comment by schwarherz** [comment]
There's a way to use the release function to set up downloads. I used to do it for CK2Plus but switched when I went to the Workshop.

**Comment by xirsoi** [comment]
I already use that :)
You can download a release based on its tag, the downloads in my signature and in the main post of the forum thread use it.

**Pull request merged**